workers/tester.sh

```diff
@@ -4,7 +4,7 @@
 
 # This script should always run as if it were being called from
 # the directory it lives in.
-script_directory=`dirname "${BASH_SOURCE[0]}"`
+script_directory=`cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd`
```

Comment:
You are right. Then what is the advantage of this revision compared to the original one?